oneview_redfish_toolkit/api/redfish_error.py

```diff
@@ -22,7 +22,7 @@
     OneViewRedfishResourceNotFoundError
```

Comment:
Copyright of this file should be updated.

Reply:
Done.